src/zig2nix/src/cli.zig

```diff
@@ -148,13 +152,18 @@ const ZigEnv = struct {
     version: []const u8,
 };
 
-pub fn fetchZigEnv(allocator: std.mem.Allocator) !std.json.Parsed(ZigEnv) {
+pub fn fetchZigEnv(allocator: std.mem.Allocator, error_writer: *std.Io.Writer) !std.json.Parsed(ZigEnv) {
     var proc = try pipe(allocator, null, &.{ "zig", "env" });
     defer proc.deinit();
     proc.close();
-    var reader = std.json.reader(allocator, proc.reader());
-    defer reader.deinit();
-    const env = try std.json.parseFromTokenSource(ZigEnv, allocator, &reader, .{ .ignore_unknown_fields = true });
+    var buf: [1024]u8 = undefined;
+    var proc_reader = proc.reader(&buf);
+    var tmp = try std.Io.Writer.Allocating.initCapacity(allocator, 1024);
+    defer tmp.deinit();
+    try @import("zon2json.zig").parse(allocator, &proc_reader.interface, &tmp.writer, error_writer, .{.file_name = "zig env"});
```

Comment:
Could perhaps use std.zon instead. Also to support older zig versions, we need to support zig env that returns json as well.